src/main/java/com/fasterxml/jackson/dataformat/xml/util/DefaultXmlPrettyPrinter.java

```diff
@@ -77,6 +77,22 @@
      */
     protected boolean _spacesInObjectEntries = true;
 
+    /**
+     * By default, will try to set as System.getProperty("line.separator").
+     * Can later set custom lineFeed with withCustomlineFeed method.
+     * @since 2.15
+     */
+    private final static String SYSTEM_DEFAULT_NEW_LINE;
+    static {
+        String lf = null;
+        try {
+            lf = System.getProperty("line.separator");
+        } catch (Throwable t) { } // access exception?
```

Comment:
Let's just catch `Exception`, not `Throwable` (ok unlike we'd get OOME or such but as a general good practice)

Reply:
late reply, but done 👍🏻